venues/ICLR.cc/2018/Conference/python/programchairs/get-acceptance-status.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+
+"""
+Dump csv file of meta-review information to make final decision.
+"""
+import argparse
+import openreview
+import csv
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+parser.add_argument('--ofile', help="output file name - default to status.csv")
+args = parser.parse_args()
+
+def init_paper_info(client):
+    print "loading submissions"
+    submissions = client.get_notes(invitation='ICLR.cc/2018/Conference/-/Blind_Submission')
+    paper_info={}
+    for n in submissions:
+        paper_info[n.number] = {}
+        paper_info[n.number]['title'] = n.content['title']
+        paper_info[n.number]['AC_rec'] = ""
+        paper_info[n.number]['AC_conf'] = ""
+        paper_info[n.number]['AC_meta'] = ""
+        paper_info[n.number]['AC_name'] = ""
+        paper_info[n.number]['ave_score'] = 0
+        paper_info[n.number]['reviews'] = []
+        paper_info[n.number]['num_review'] = 0
+    return paper_info
+
+def get_activity(client, paper_info):
+    # load in all notes from conference
+    print "loading reviews"
+    limit = 1000
+    offset = 0
+    notes = []
+    batch = client.get_notes(invitation="ICLR.cc/2018/Conference/-/.*", limit=limit, offset=offset)
```

Comment:
to avoid getting all the notes you can add the word 'Review' to the regex? Does it work?